### PR TITLE
Proxy client move#16513 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

https://github.com/SUSE/spacewalk/issues/16513

### CODE REVIEW

**Comment by keichwa** [comment]
@nadvornik if time permits, please help with reviewing this PR.  TIA!

**Comment by keichwa** [comment]
I'll submit the .changelog entry later.

**Review by jcayouette — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/client-proxy.adoc [hunk 1]
```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
...
+== Move Clients between Proxies
+
+You can move Salt and Salt SSH push clients between proxies without the need to repeat the registration process.
+
+
+
+[NOTE]
+====
+If you want to move a traditional client between proxies or any client between chained proxies you must repeat the registration process from the beginning.
+====
```
Comment (nadvornik) [inline modules/client-configuration/pages/client-proxy.adoc:36]: This is not correct.
Moving salt client between chained proxies is supported.
Moving chained proxy server is not supported even if it is managed by salt. Proxy server must always registered from the beginning.

File: modules/client-configuration/pages/client-proxy.adoc [hunk 2]
```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
...
+
+// Procede as follows:
+
+
+
+.Procedure: Moving Salt or  Salt SSH Push Client between Proxies
+
+. In the {productname} {webui}, navigate to the [guimenu]``System Details`` page for the client you want to move between proxies.
+. Navigate to the [guimenu]``Connection`` tab.
```
Comment (nadvornik) [inline modules/client-configuration/pages/client-proxy.adoc:45]: Then follow "Change proxy" link to see the drop-down menu.

File: modules/client-configuration/pages/client-proxy.adoc [hunk 3]
```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
...
+
+The function updates the proxy path immediately in the database and new action
+for refreshing the channel URLs is scheduled.
+
+
+
+== Move Clients from Proxies to the Server
+
+If you want to move any Salt or traditional client from proxies to the server you must repeat the registration process from the beginning.
+
```
Comment (nadvornik) [inline modules/client-configuration/pages/client-proxy.adoc:90]: Moving salt client from proxy to server is supported, just select "None" from proxy list.

**Review by nadvornik — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/client-proxy.adoc [hunk 4]
```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
...
+== Move Clients between Proxies
+
+You can move Salt and Salt SSH push clients between proxies without the need to repeat the registration process.
+
+
+
+[NOTE]
+====
+If you want to move a traditional client between proxies or any client between chained proxies you must repeat the registration process from the beginning.
```
Comment (keichwa) [inline modules/client-configuration/pages/client-proxy.adoc:35]:
```suggestion
If you want to move a traditional client between proxies you must repeat the registration process from the beginning.
```

File: modules/client-configuration/pages/client-proxy.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/client-proxy.adoc:36]: ok, thanks.  I got it wrong.  I think we can simplify it.

File: modules/client-configuration/pages/client-proxy.adoc (hunk 2 above)
Comment (keichwa) [inline modules/client-configuration/pages/client-proxy.adoc:45]:
```suggestion
. Navigate to the [guimenu]``Connection`` tab.
  Then follow the [guimenu]``Change proxy`` link to see the drop-down menu.
```

File: modules/client-configuration/pages/client-proxy.adoc [hunk 5]
```diff
@@ -21,4 +22,69 @@ You can see a list of clients that are connected to a proxy by clicking the name
...
+==== Salt SSH Push Clients
+
+The function updates the proxy path immediately in the database and new action
+for refreshing the channel URLs is scheduled.
+
+
+
+== Move Clients from Proxies to the Server
+
+If you want to move any Salt or traditional client from proxies to the server you must repeat the registration process from the beginning.
```
Comment (keichwa) [inline modules/client-configuration/pages/client-proxy.adoc:89]:
```suggestion
If you want to move a Salt client from a proxy to the server, select [literal]``None`` from proxy list.

If you want to move a traditional client to the server you must repeat the registration process from the beginning.
```

File: modules/client-configuration/pages/client-proxy.adoc (hunk 3 above)
Comment (keichwa) [reply modules/client-configuration/pages/client-proxy.adoc:90]: ok

**Review by keichwa — COMMENTED** [review]
thank you!

**Comment by keichwa** [comment]
I hope I got it right now.

**Review by nadvornik — APPROVED** [review]
LGTM

**Comment by jordimassaguerpla** [comment]
@0rnela This could be a way to migrate proxies from a previous version. Yesterday I asked at the Team Review and I understood we could use this to migrate salt clients.

Traditional clients can't be upgraded using this.

Chained proxies can't be upgraded using this. A chain proxy would be a proxy that is registered to another proxy. In this case, the user should install from scratch all the proxy chain. Then, he could migrate the clients using this procedure.

@0rnela I am telling you because you might want to review the documentation on how to upgrade a proxy to see if this could be an alternative. In that case, we could simple add a link from the upgrade docs to this one, or?

**Comment by jordimassaguerpla** [comment]
@mbussolotto jfyi ^

**Pull request merged**